apps/api/validators.py

```diff
@@ -1,12 +1,384 @@
-from django.core.exceptions import ValidationError
+'''Validates if the file given is a csv file'''
 
 import csv
+from re import match
+from django.core.exceptions import ValidationError
+
+
+class ColumnError(Exception):
+    '''Exception raised if the number of columns in the csv file header does
+    not match the rest'''
+    pass
+
+
+class MultipleUtoridOccurences(Exception):
+    '''Exception raised if the same utor id occurs multiple times in a file
+    '''
+    pass
+
+
+class InvalidUtorid(Exception):
```

Comment:
Same as ColumnError. Always passing in current row number and same message.

Possible solution:

class InvalidUtorid(Exception):
		def __init__(self, line_number):
			msg = ‘Invalid utorid at row: {}, column: 0’.format(line_number)
			super(InvalidUtorid, self).__init__(msg)

Ex. raise InvalidUtorid(4)
Invalid utorid at row: 4, column: 0’